counter/package.json

```diff
@@ -1,29 +1,24 @@
 {
     "scripts": {
-        "start": "npx webpack-dev-server",
-        "build": "npx webpack"
+        "postinstall": "bower install bower.json && bower install",
+        "start": "pulp build && concurrently \"pulp --watch build\" \"parcel index.html\""
     },
     "devDependencies": {
         "@babel/core": "^7.4.4",
-        "@babel/preset-env": "^7.4.4",
         "@babel/preset-react": "^7.0.0",
```

Comment:
Do we need any of the babel references?

Reply:
I was sure I removed them all. Maybe missed some, will take a look